zerver/views/user_groups.py

```diff
@@ -202,9 +202,6 @@ def notify_for_user_group_subscription_changes(
         if recipient_user.is_bot:
             # Don't send notification message to bots.
             continue
-        if not recipient_user.is_active:
-            # Don't send notification message to deactivated users.
-            continue
```

Comment:
Probably leaving an `assert not recipient_user.is_active` here would be a good idea, with a comment..